modules/ROOT/pages/organization.adoc

```diff
@@ -38,23 +38,28 @@ An Anypoint Platform user who has been granted organization administrator privil
 On the *Organization* page, you can:
 
 * View a hierarchical tree of all of the organizations you have permissions to view. You can view and edit properties of an organization by clicking the name.
-* Click the name of a group to view and edit information about it. What you can view and edit depends on your role. +
+* Click the name of a group to view and edit information about it. What you can view and edit depends on your role.
+
 [NOTE]
 --
-*Changing the name or domain name of an organization changes the deep links to any existing API Portals in your organization.
-* Add and delete business groups (if enabled).
+Changing the name or domain name of an organization changes the deep links to any existing API Portals in your organization.
 --
 
+* Add and delete business groups (if enabled).
+
 === Access the Organization page:
 
 . Sign into your Anypoint Platform account as a user with the Organization Administrators role.
-. Select the *Access Management* option from the main page, or select it from the left side bar by clicking the _menu_ button in the top left corner. +
+. Select the *Access Management* option from the main page, or select it from the left side bar by clicking the _menu_ button in the top left corner.
++
+image::anypoint-platform-nav-menu.png[]
++
 The *Organization* page is displayed, where you can view organizations and business groups and add new business groups (if enabled). +
 You can also click *Organization* in the left menu to view the Organization page.
 
 === Client ID and Client Secret
 
-Each environment and business group in an organization has a unique client ID and client secret that are used for authentication. You can use these to access assets within an organization if you are not an organization administrator. The client ID and password are generated by Anypoint Platform for each environment that you create, and they are globally unique.
+Each environment and business group in an organization has an associated unique client ID and client secret. These are used for authentication by users who are not organization administrators to access assets within an organization. The client ID and password are generated by Anypoint Platform for each environment that you create, and they are globally unique.
```

Comment:
Should all this info about business groups be moved to the Business Groups section?

Reply:
Also, every master org has a unique client ID and client secret, yes?